src/music21/meter.ts

```diff
@@ -250,9 +256,11 @@ export class TimeSignature extends base.Music21Object {
                     beamType = 'partial-right';
                 }
             } else if (isLast) {
-                beamType = 'start';
+                beamType = 'stop';
```

Comment:
Without this, subdivisions of the last beat in a measure do not get beamed. (See https://github.com/cuthbertLab/music21/blob/master/music21/meter.py#L3755).

Reply:
Ah!  A real error!